src/utils/dmo-analytics-logic.ts

```diff
@@ -1,6 +1,10 @@
 import { dmoUrls } from '../constants';
 
-const currentURL = window.location.href;
+let currentURL = '';
+
+if (typeof window !== 'undefined') {
+    currentURL = window.location.href;
```

Comment:
Hello Mike, This has nothing to do with canonical tags, right?

Just asking because the canonical tag doesn't always match with the current URL

Reply:
No I don't think so, this is only used in the Link component to fire off CMS referral events when you head into the DMS 
![image](https://github.com/user-attachments/assets/fa628b3a-6aad-4e49-8bb4-7bc63c220d29)